Jenkinsfile

```diff
@@ -67,6 +67,27 @@ pipeline {
         }
       }
     }
+
+    stage ('Publish to rubygems.org') {
+      environment {
+        GEM_HOST_API_KEY = credentials('jenkins-rubygems-api-key')
+      }
+      when {
+        anyOf {
+          branch 'master'
```

Comment:
Not on master.

Reply:
@blambeau ok so when do we want it to be triggered ?